docs/content/intro/quickstart.md

```diff
@@ -2,43 +2,265 @@
+$env:PATH+=";$env:USERPROFILE\bin"
 ```
 
-You will now have Brigade installed. [Kashti](https://github.com/brigadecore/kashti), the dashboard for your Brigade pipelines, is also installed in the cluster.
+The script above downloads brig.exe and adds it to your PATH for the current session.
+Add the following line to your [PowerShell Profile](https://www.howtogeek.com/126469/how-to-create-a-powershell-profile/) to make the change permanent.
 
-## Install brig
+```powershell
+$env:PATH+=";$env:USERPROFILE\bin"
+```
 
-Brig is the Brigade command line client. You can use `brig` to create/update/delete new brigade Projects, run Builds, etc. To get `brig`, navigate to the [Releases page](https://github.com/brigadecore/brigade/releases/) and then download the appropriate client for your platform. For example, if you're using Linux or WSL, you can get the 1.2.1 version in this way:
+## Install Brigade
 
-```bash
-wget -O brig https://github.com/brigadecore/brigade/releases/download/v1.2.1/brig-linux-amd64
-chmod +x brig
-mv brig ~/bin
+Install Brigade on your local development cluster. See our [Installation] instructions for full instructions suitable for production clusters.
+
+1. Enable Helm's experimental OCI support by setting the `HELM_EXPERIMENTAL_OCI` environment variable to 1.
+
+    **bash**
+    ```bash
+    export HELM_EXPERIMENTAL_OCI=1
+    ```
+
+    **powershell**
+    ```powershell
+    $env:HELM_EXPERIMENTAL_OCI=1
+    ```
+
+1. Create a directory to store the Brigade Helm charts.
+
+    **bash**
+    ```bash
+    mkdir -p ~/charts
+    ```
+
+    **powershell**
+    ```powershell
+    mkdir -force $env:USERPROFILE/charts
+    ```
+
+1. Run the following commands to install Brigade.
+
+    ```
+    helm chart pull ghcr.io/brigadecore/brigade:v2.0.0-alpha.5
+    helm chart export ghcr.io/brigadecore/brigade:v2.0.0-alpha.5 -d ~/charts
+    kubectl create namespace brigade2
+    helm install brigade2 ~/charts/brigade --namespace brigade2
+    kubectl rollout status deployment brigade2-apiserver -n brigade2 --timeout 5m
+    ```
+    
+    Wait for the Brigade deployment to be ready.
+    If the deployment fails, proceed to the [installation troubleshooting](/intro/install/#troubleshooting) section.
+
+1. Make the Brigade API server available using port forwarding. This is necessary only for development clusters that do not have a public IP address.
+
+    **bash**
+    ```
+    kubectl --namespace brigade2 port-forward service/brigade2-apiserver 8443:443 &>/dev/null &
+    ```
+
+    **powershell**
+    ```
+    & kubectl --namespace brigade2 port-forward service/brigade2-apiserver 8443:443 *> $null  
+    ```
+
+[Installation]: /intro/install/
+
+## Log in to Brigade
+
+Authenticate to Brigade as the root user using demo password `F00Bar!!!`. The -k flag instructs Brigade to ignore the self-signed certificate used by our local installation of Brigade.
+
+```
+brig -k login --server https://localhost:8443 --root
+```
```

Comment:
As of alpha.5 (or maybe alpha.4... I can't remember), all flags _follow_ the subcommand and none ever, ever appear before it.

Also, `-k` is only applicable to `brig login` now. We remember the choice to ignore cert issues and `-k` has been dropped from all other commands.

Reply:
Great! Thanks I had written this part on alpha.4 and hadn't noticed the change yet.